ArduPlane/ArduPlane.cpp

```diff
@@ -76,6 +76,7 @@ const AP_Scheduler::Task Plane::scheduler_tasks[] = {
     SCHED_TASK(Log_Write_Fast,         25,    300),
     SCHED_TASK(update_logging1,        10,    300),
     SCHED_TASK(update_logging2,        10,    300),
+    SCHED_TASK(update_soaring,         50,    400),
```

Comment:
This may have changed since the port so I though id mention this. The 50 here is in Hertz. Is that the intent? I see rate_hz is set to 5. If you want it to run at 5hz then this value here should be 5.